opentelemetry-sdk/src/propagation/trace_context.rs

```diff
@@ -33,8 +17,25 @@ const TRACESTATE_HEADER: &str = "tracestate";
 static TRACE_CONTEXT_HEADER_FIELDS: Lazy<[String; 2]> =
     Lazy::new(|| [TRACEPARENT_HEADER.to_owned(), TRACESTATE_HEADER.to_owned()]);
 
-/// Propagates `SpanContext`s in [W3C TraceContext] format.
+/// Propagates `SpanContext`s in [W3C TraceContext] format under `traceparent` header.
```

Comment:
its best to simply link to W3CTraceContext spec here?
Also, `traceparent` + `tracestate` both together makes the spec. This doc comment seems to indicate that `traceparent` alone is present..

Reply:
> its best to simply link to W3CTraceContext spec here?

It should be linked already. The link is set on L39